From 0c778a854d4fd2c86059604ae13bf9bc9c3e1296 Mon Sep 17 00:00:00 2001
From: HungTran <hung.tran.jy@rvc.renesas.com>
Date: Wed, 9 Dec 2015 03:46:31 +0700
Subject: [PATCH] H264Streaming: Fix unstable connect for External device

When external device connect, rtsp-server must wait for the config info.
This config-info is resent after a time set by "config-interval" property.
Reduce this config-interval can make external device connect more stable
(if set 3s, rtsp-server must wait for 1, 2, or 3s before get config info)

Signed-off-by: HungTran <hung.tran.jy@rvc.renesas.com>
---
 src/plugins/gstreamer/H264Streaming/qgstreamerH264Streamingsession.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/gstreamer/H264Streaming/qgstreamerH264Streamingsession.cpp b/src/plugins/gstreamer/H264Streaming/qgstreamerH264Streamingsession.cpp
index a4f5025..80552ea 100755
--- a/src/plugins/gstreamer/H264Streaming/qgstreamerH264Streamingsession.cpp
+++ b/src/plugins/gstreamer/H264Streaming/qgstreamerH264Streamingsession.cpp
@@ -651,7 +651,7 @@ void QGstreamerH264StreamingSession::componentComplete()
 				"udpsrc port=5000 caps=\"application/x-rtp\" ! "
 				"rtph264depay ! h264parse ! "
 				"video/x-h264,stream-format=avc,alignment=au ! "
-				"queue leaky=2 ! rtph264pay name=pay0 pt=96 config-interval=3 mtu=9000 "
+				"queue leaky=2 ! rtph264pay name=pay0 pt=96 config-interval=1 mtu=9000 "
 				")"
 			);
 
-- 
1.9.1

